docs/compass/08-01-basic-flow-tutorial.md

```diff
@@ -45,27 +45,36 @@ For simplicity reasons, use the available Order Service as the sample external A
 
 ### Use your Application in the Kyma Console UI
 
-1. Go back to the Kyma Console UI. You can see that the `test-app` Application is registered in the **Applications** view. Select `test-app` and bind it to your Namespace by selecting the **Create Binding** button.
+1. Go back to the Kyma Console UI. You can see that the `test-app` Application is registered in the **Applications/Systems** view. Select `test-app` and bind it to your Namespace by selecting the **Create Binding** button.
 
-2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package and clicking the **Add** button in the top-right corner of the page.
+2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package as selected plan and clicking the **Add** button in the top-right corner of the page.
```

Comment:
```suggestion
2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package as selected plan and clicking the **Add** button in the top-right corner of the page.
```
`as selected plan` -> what do you mean?

Reply:
There is a dropdown called "Selected plan". From here you have to select your Package.

Reply:
```suggestion
2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package as **Selected plan** from the dropdown list and clicking the **Add** button in the top-right corner of the page.
```
Or you can skip the `from the dropdown list` if you think it's too lengthy.